src/ocp_postprocess.rs

```diff
@@ -59,6 +59,8 @@ pub(crate) async fn ocp_postprocess(
 
     run_cluster_customizations(cluster_customizations, in_memory_etcd_client).await?;
 
+    set_cluster_version_available_false(in_memory_etcd_client).await?;
```

Comment:
```suggestion
    // We do this because ...
    set_cluster_version_available_false(in_memory_etcd_client).await?;
```

Reply:
Good point, I'll talk with Igal to verify the reason and update this change with a proper comment.

Reply:
Currently it is very difficult to say when we finish installation and when data is not stale. 
Right now looking on cvo status does nothing as cvo doesn't even update this condition while starting in ibi/ibu.
This change allows us to monitor cluster installation with regular tools like looking on cvo available condition that will be set back to True when cvo will validate that everything is ok

Reply:
@omertuc updated, PTAL.